lua/supermaven-nvim/commands.lua

```diff
@@ -43,6 +43,22 @@ M.setup = function()
   vim.api.nvim_create_user_command("SupermavenClearLog", function()
     api.clear_log()
   end, {})
+
+  vim.api.nvim_create_user_command("SupermavenDisable", function()
+    api.stop()
+    vim.cmd([[
+      let g:SUPERMAVEN_DISABLED = 1
```

Comment:
Is there any particular reason we are using a vim command here instead of setting `vim.g.SUPERMAVEN_DISABLED = 1`?

Reply:
the standard lua syntax with vim.g.* wasn't working for some reason. Should I stick to using that